torch/utils/benchmark/examples/blas_compare_setup.py

```diff
@@ -114,6 +114,7 @@ def main():
             f"source activate {env_path}",
             shell=True,
             capture_output=True,
+            check=True,
```

Comment:
Should be False, shouldn't it, as error code is checked on the next line?
```suggestion
            check=False,
```